test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ApiBehaviorApplicationModelProviderTest.cs

```diff
@@ -493,48 +513,55 @@ public void AddMultipartFormDataConsumesAttribute_AddsConsumesAttribute_WhenActi
             Assert.Equal("multipart/form-data", Assert.Single(consumesAttribute.ContentTypes));
         }
 
-        private static ApiBehaviorApplicationModelProvider GetProvider(
-            ApiBehaviorOptions options = null,
-            IModelMetadataProvider modelMetadataProvider = null)
+        private ApiBehaviorApplicationModelProvider GetProvider(
+            ApiBehaviorOptions options = null)
         {
             options = options ?? new ApiBehaviorOptions
             {
                 InvalidModelStateResponseFactory = _ => null,
             };
             var optionsAccessor = Options.Create(options);
 
-            modelMetadataProvider = modelMetadataProvider ?? new TestModelMetadataProvider();
             var loggerFactory = NullLoggerFactory.Instance;
-
+            var modelMetadataProvider = TestModelMetadataProvider.CreateDefaultProvider();
             return new ApiBehaviorApplicationModelProvider(optionsAccessor, modelMetadataProvider, loggerFactory);
         }
 
-        private static ApplicationModelProviderContext GetContext(Type type)
+        private ApplicationModelProviderContext GetContext(Type type)
```

Comment:
Why are these helper methods no longer static? Looks like change for change sake.